lighthouse-plugin-publisher-ads/audits/script-injected-tags.js

```diff
@@ -67,6 +67,7 @@ const STATICALLY_LOADABLE_TAGS = [
   /static\.criteo\.net\/js\/.*\/publishertag\.js/,
   /www\.googletagservices\.com\/tag\/js\/gpt\.js/,
   /pagead2\.googlesyndication\.com\/pagead\/js\/adsbygoogle\.js/,
+  /cdn\.ampproject\.org\/v0\/amp-ad-0\.1\.js/,
```

Comment:
Do you anticipate the version numbers changing at any point in the foreseeable future? Might be better to keep the matching more generic. In `resource-classification.js` as well.

Reply:
AFAIK no. My understanding is that versions were part of the AMP plugin design, but never really happened

Reply:
made this a bit more robust with regex